_includes/list-partners.html

```diff
@@ -0,0 +1,19 @@
+{% if include.style == "list" %}
+  <ul>
+    {% for agency in project.partner %}
+      {% capture agencyname %}{{ agency | slugify }}{% endcapture %}
+      {% capture partnerurl %}{% for partner in site.data.partners %}{% if partner[0] == agencyname %}{{ partner[1].url }}{% else %} {% endif %}{% endfor %}{% endcapture %}
```

Comment:
I'd suggest just looping through `{% for agency in project.partner %}` to grab the captures before the conditional. It _shouldn't_ have a noticeable negative impact on rendering speed. Should render the loops that build the different output styles clearer, too.